MANIFEST.in

```diff
@@ -6,5 +6,5 @@ include README.md
 recursive-include tests *
 recursive-exclude * __pycache__
 recursive-exclude * *.py[co]
-
+recursive-include *.txt
```

Comment:
I'm fairly new to these MANIFEST files. Is the first asterisk optional? Does it make any difference to have this on a separate line from the `.rst` & `.md` files?